### ISSUE TITLE
The paper and the code does not match 

### ISSUE THREAD

**Issue by LyazS** [bigvideoresearch/OIM#2]
The OIM in paper use variable _alpha_ and the IR loss in paper use _beta_
but in code, 
```alpha = np.array([float(os.getenv('alpha', None))])```
```beta = np.array([float(os.getenv('beta', None))])```
and nowhere to find to set the variable. Maybe they are not the same thing?

And the IR loss in paper is that: _L=1/P * w * y * (1+z) * logx_
but in the code **./OIM/caffe-oim/src/caffe/layers/weighted_softmax_loss_layer.cu**
it run in another way.

Could u please explain it or release the loss in pytorch?

**Comment by Yinyf0804** [comment]
Maybe alpha, beta are got from train_oim.sh, and they both have the same meaning with beta in paper. The ratio in code means alpha in paper.

I have a question in ./OIM/caffe-oim/src/caffe/layers/weighted_softmax_loss_layer.cu too.  Why the code uses (-beta) in line 97 but uses (1.0 + alpha)  in line 104.  Besides, I'm not understand why the condition in line94 can suppress background proposals.

I will be grateful if you can give me some advice.

**Comment by chenhaolin1989** [comment]
Yes, the naming of parameters in the code have a small different with those in the paper.  
ratio (in the code) = alpha (in the paper). alpha,beta (in the code) = beta (in the paper, one for surrounding proposal and one for center proposal). 
Thanks for pointing out the -beta problem. It should be beta. 
We suppress the center proposal by making the gradients of surrounding proposals scale up to (1 + beta) of its original value, and gradient of the center proposal scale to beta.

**Comment by LyazS** [comment]
> 
> 
> Yes, the naming of parameters in the code have a small different with those in the paper.

Could u please point it out clearly？
